plone/scale/scale.py

```diff
@@ -50,6 +50,13 @@ def scaleImage(image, width=None, height=None, direction='down',
 
     image = scalePILImage(image, width, height, direction)
 
+    # convert to simpler mode if possible
+    if image.mode not in ('P', 'L') and image.getcolors(maxcolors=256):
+        if format_ == 'JPEG':
```

Comment:
The test for grayscale is missing for JPEG, any JPEG with less then 256 colors is converted to greyscale at the moment.